charts/turbinia/README.md

```diff
@@ -0,0 +1,322 @@
+<!--- app-name: Turbinia -->
+# Turbinia Helm Chart
+
+Turbinia is an open-source framework for deploying, managing, and running distributed forensic workloads. 
+
+[Overview of Turbinia](https://turbinia.readthedocs.io/en/latest/)
+
+[Chart Source Code](https://github.com/google/osdfir-infrastructure)
+## TL;DR
+
+```console
+helm install turbinia-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia
+```
+> **Tip**: To quickly get started with a local cluster, see [minikube install docs](https://minikube.sigs.k8s.io/docs/start/).
+
+## Introduction
+
+This chart bootstraps a [Turbinia](https://github.com/google/turbinia/blob/master/docker/release/build/Dockerfile-latest) deployment on a [Kubernetes](https://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- PV provisioner support in the underlying infrastructure
+
+> **Note**: Currently Turbinia only supports processing of GCP Persistent Disks and Local Evidence. See [GKE Installations](#gke-installations) for deploying to GKE.
+
+## Installing the Chart
+
+To install the chart, specify any release name of your choice. For example, if you
+want to install the chart for development, you can choose a release name of `turbinia-dev` then run:
+```console
+helm install turbinia-release oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia
+```
+
+The command deploys Turbinia on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured 
+during installation or see [Installating for Production](#installing-for-production) 
+for a recommended production installation. 
+
+> **Tip**:  You can override the default Turbinia configuration by placing the 
+`turbinia.conf` config at the root of the Helm chart. When choosing this option, 
+pull and install the Helm chart locally.
+
+## Installing for Production
+
+Pull the chart locally and review the `values-production.yaml` file for a list of values that will be used for production.
+```console
+helm pull oci://us-docker.pkg.dev/osdfir-registry/osdfir-charts/turbinia --untar
+```
+
+### GKE Installations
+Create a Turbinia GCP account using the helper script in `tools/create-gcp-sa.sh` prior to installing the chart. 
```

Comment:
full path?

Reply:
Per chat not needed anymore or this would still be tools/create-gcp-sa.sh if the Turbinia helm chart was pulled instead